Example/Emission/AuthenticationManager.m

```diff
@@ -41,7 +42,9 @@ - (void)presentAuthenticationPromptOnViewController:(UIViewController *)viewCont
 
 - (void)showAuthenticationToArtsy:(NSString *)errorMessage completion:(dispatch_block_t)completion
 {
-  NSString *message = errorMessage ? errorMessage : @"Enter your Artsy credentials";
+  AppSetup *setup = [AppSetup ambientSetup];
+  NSString *prodOrStaging = setup.inStaging ? @"staging" : @"production";
+  NSString *message = errorMessage ? errorMessage : [NSString stringWithFormat:@"Enter your %@ Artsy credentials", prodOrStaging];
   UIAlertController *alert = [UIAlertController alertControllerWithTitle:@"Authentication"
```

Comment:
Now easier to know up-front what the current state is:

![screen shot 2017-09-22 at 11 50 22 am](https://user-images.githubusercontent.com/49038/30754657-864589e0-9f91-11e7-9770-632cf409bbe4.png)